scripts/install.sh

```diff
@@ -80,6 +80,9 @@ if [[ "$DISTRO" = "Ubuntu" && "$INSTALL_NETWORK_MANAGER" != "true" && -z "$QUIET
   fi
 fi
 
+echo "Update package list"
+apt-get update
```

Comment:
Are you sure we don't do this earlier?

Reply:
No clue. I just put in in for good measure because I saw the jre install explicitly did it for itself.